packages/core/src/navigation-item/NavigationItem.tsx

```diff
@@ -1,14 +1,60 @@
+import React, {
+  AriaAttributes,
+  ForwardedRef,
+  MouseEvent,
+  ReactElement,
+} from "react";
 import { ComponentPropsWithoutRef, forwardRef, MouseEventHandler } from "react";
 import { makePrefixer } from "../utils";
 import { clsx } from "clsx";
 import { ExpansionIcon } from "./ExpansionIcon";
-import { ConditionalWrapper } from "./ConditionalWrapper";
 
 import { useWindow } from "@salt-ds/window";
 import { useComponentCssInjection } from "@salt-ds/styles";
 
 import navigationItemCss from "./NavigationItem.css";
 
+type OptionalPartial<T, K extends keyof T> = Partial<Pick<T, K>>;
+
+export interface NavigationItemElementProps<
+  T extends HTMLAnchorElement | HTMLButtonElement
+> extends Pick<
+    AriaAttributes,
```

Comment:
I don't think aria attributes needs to be picked out, if those are not working, some other types is not working?